### PR TITLE
The Web Content Accessibility Guidelines (WCAG) lesson: fixed some found typos

### PR DESCRIPTION
**Pull request by bappyasif**
found a couple of typos and this commit has fixes them

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX
this pr fixes a couple of found typos in this document [The Web Content Accessibility Guidelines (WCAG)](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-the-web-content-accessibility-guidelines-wcag)

**2. This PR:**

resolves these typos:
* adjusted "capitalization" of a word
* used an appropriate auxiliary verb

**3. Additional Information:**

### CODE REVIEW

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Conformance levels get mentioned at a couple of other points in these lessons, a
 
 ### (Before) Implementing Accessibility
 
-The lessons that follow dive into several concepts that can help you improve the accessibility of your websites, but they don't cover *every* facet of accessibility. The goal of these lessons is just to introduce you to some of the more common or simpler concepts that you should get into the habit of checking for moving forward.
+The lessons that follow dive into several concepts that can help you improve the accessibility of your websites, but they don't cover *every* facet of accessibility. The goal of these lessons are just to introduce you to some of the more common or simpler concepts that you should get into the habit of checking for moving forward.
```
Comment (dm-murphy) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:40]: The use of "are" would be incorrect here. "Are" is a plural verb and "goal" is actually the singular subject. "Lessons" is not the subject here if that is what confused you.

In the future, try to only make these types of changes if you are certain your suggestion is correct. You can also use a service like Grammarly to double check yourself. Thanks!

Comment (dm-murphy) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:40]: The use of "are" would be incorrect here. "Are" is a plural verb and "goal" is actually the singular subject. "Lessons" is not the subject here if that is what confused you.

In the future, try to only make these types of changes if you are certain your suggestion is correct. You can also use a service like Grammarly to double check yourself. Thanks!

Comment (dm-murphy) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:40]: The use of "are" would be incorrect here. "Are" is a plural verb and "goal" is actually the singular subject. "Lessons" is not the subject here if that is what confused you.

In the future, try to only make these types of changes if you are certain your suggestion is correct. You can also use a service like Grammarly to double check yourself. Thanks!

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ It's important to understand that while the WCAG can be incredibly helpful for b
 
 The WCAG are organized around four, core principles that should be kept in mind when implementing any sort of accessibility feature:
 
-1. **Perceivable:** Users must be able to perceive the information or user interfaces being presented. For example, Light text on a light background could be difficult for some users with a visual impairment to perceive.
+1. **Perceivable:** Users must be able to perceive the information or user interfaces being presented. For example, light text on a light background could be difficult for some users with a visual impairment to perceive.
```
Comment (dm-murphy) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:22]: Nice catch on the capitalization here, thanks!

Comment (dm-murphy) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:22]: Nice catch on the lack of capitalization here, thanks!

**Review by dm-murphy — CHANGES_REQUESTED** [review]

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md (hunk 1 above)
Comment (bappyasif) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:40]: thanks, i'll do that :)

please feel free to discard this change and merge with what seems right to you :)

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md (hunk 2 above)
Comment (bappyasif) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:22]: i have updated this patch with your requested review changes, please let me know if there is anymore changes required, thanks :)

Comment (dm-murphy) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:22]: Thanks ab!

**Pull request merged**